src/MongoDB.Driver.Core/ChangeStreamUpdateDescription.cs

```diff
@@ -35,12 +37,27 @@ public sealed class ChangeStreamUpdateDescription
         /// </summary>
         /// <param name="updatedFields">The updated fields.</param>
         /// <param name="removedFields">The removed fields.</param>
+        [Obsolete("Use the overload with truncatedArrays.")]
```

Comment:
Is it really Obsolete?

Isn't the new truncateArrays argument just optional? (For example: this class is used with older servers that do NOT include this field).

Reply:
As far as I can see, UpdateDescription will always have this new field. So, all new server versions won't have a case when this value is not presented. However, all previous servers still don't know about this value, so to support them we still have the previous constructor but which is marked as obsolete. I don't have a strong opinion here and can choose any way :)

Reply:
I would say this constructor is not Obsolete because it is still required for some server versions and will be for a long time.

Reply:
Ok, done